source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -62,6 +62,8 @@ class CAdvisorMetricsAPIClient
   @@telemetryCpuMetricTimeTracker = DateTime.now.to_time.to_i
   @@telemetryMemoryMetricTimeTracker = DateTime.now.to_time.to_i
   @@telemetryPVKubeSystemMetricsTimeTracker = DateTime.now.to_time.to_i
+  @@userAgent = "ama-logs-" + (ENV["AGENT_VERSION"].nil? ? "" : ENV["AGENT_VERSION"])
```

Comment:
(1) I think,  we should follow standard naming convention of the user agent  https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent.  In this case, ama-logs/<AgentVersion> instead of ama-logs-<AgentVersion>
(2) should we include the OS information also, for example ama-logs/<AgentVersion>(Linux)  or ama-logs/<AgentVersion>(Windows) ?